labs/docker/coturn/turnserver.conf.tmpl

```diff
@@ -1,14 +1,36 @@
 listening-port={{ .Env.PORT }}
-min-port=49152
-max-port=65535
+tls-listening-port={{ .Env.TURN_PORT }}
+
+# Fingerprints in TURN messages are required for WebRTC
 fingerprint
+
+# The long-term credential mechanism is required for WebRTC
 lt-cred-mech
```

Comment:
*static-auth-secret* implies *use-auth-secret* (and sets: ignore realms and credentials in the DB) which in turn implies *lt-cred-mech* (and enables time-limited credentials). So **static-auth-secret** is sufficient.